src/test/kotlin/com/beust/klaxon/Serialization.kt

```diff
@@ -0,0 +1,62 @@
+package com.beust.klaxon
```

Comment:
You should call this `SerializationTest`, just to be safe with all build tools. I'll merge after that.

Thanks!